Resources/Prototypes/Entities/Mobs/Player/human.yml

```diff
@@ -48,3 +48,181 @@
     description: Inspect the station, jot down performance reviews for heads of staff, bug the Captain.
   - type: UtilityAI
     startingGear: CentcomGear
+
+# ERT Leader
+- type: entity
+  name: ERT leader
+  parent: MobHuman
+  id: MobHumanERTLeader
+  components:
+  - type: Icon
+    sprite: Markers/jobs.rsi
+    state: ertleader
+  - type: GhostRoleMobSpawner
+    prototype: MobHumanERTLeader
+    deleteOnSpawn: true
+    makeSentient: false
+    name: ERT Leader
+    description: Lead a team of specialists to resolve the stations issues.
+  - type: UtilityAI
+    startingGear: ERTLeaderGear
```

Comment:
Just make a new component or something at least please.

Reply:
These are takeovers yes for admins to spawn. There are spawners too for use with mapped shuttles and for events firing in future.

The UtilityAI doesn't allow movement on takeover hence the tick box issue, making a component for it is probably (very much likely) out of my ability. So at least this is just a big content PR.